flow/metrics.yaml

```diff
@@ -30,6 +30,17 @@ components:
             Latency metrics.
           $ref: '#/components/schemas/Flow.Latency.Metrics'
           x-field-uid: 4
+        egress:
```

Comment:
Another comment related to how user sees results: Say we have a flow 10 sub-flows , 8 have MPLS header and 2 dont have MPLS header and user does egress tracking by MPLS 3 bit TOS/EXP bits . It looks like as per the example, even the 2 sub-flows which dont have MPLS header at all will get added to some bucket arbitrarily based on values in the 3 bits where MPLS EXP would have been if it had been present. Isn't this behaviour definition problematic  ? Should it not be that if packet does not confirm to packet shape as defined ( excluding custom ) , only those packets should be filtered from aggregate flow which matches the defined egress packet shape ?